### PR TITLE
Problem: cmd append with existing arguments in test_tx_inclusion

### PR DESCRIPTION
**Pull request by mmsqe**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: integration_tests/test_basic.py [hunk 1]
```diff
@@ -677,9 +680,16 @@ def test_tx_inclusion(cronos, max_gas_wanted):
...
+    def fn(cmd):
+        global origin_cmd
+        if origin_cmd is None:
+            origin_cmd = cmd
+        return f"{origin_cmd} --evm.max-tx-gas-wanted {max_gas_wanted}"
+
     modify_command_in_supervisor_config(
         cronos.base_dir / "tasks.ini",
-        lambda cmd: f"{cmd} --evm.max-tx-gas-wanted {max_gas_wanted}",
+        lambda cmd: fn(cmd),
```
Comment (yihuang) [inline integration_tests/test_basic.py:692]: What problem does it solve?
Do you mean when the test case run multiple times, the flags get appended multiple times?

Comment (mmsqe) [reply integration_tests/test_basic.py:692]: [yes](https://github.com/crypto-org-chain/cronos/pull/901#discussion_r1125779249)

File: integration_tests/test_basic.py [hunk 2]
```diff
@@ -669,6 +669,9 @@ def test_contract(cronos):
     assert "world" == greeter_call_result
 
 
+origin_cmd = None
+
+
```
Comment (yihuang) [inline integration_tests/test_basic.py:674]:
```suggestion
```

I think we don't need this, to keep it local.

**Review by yihuang — APPROVED** [review]
LGTM, nit: don't need to actually declare `origin_cmd` at module, since it's an implementation detail.

File: integration_tests/test_basic.py (hunk 2 above)
Comment (yihuang) [reply integration_tests/test_basic.py:674]: I see, it's needed for the initial value.

**Pull request merged**